docs/getting-started/os-support.rst

```diff
@@ -27,38 +25,18 @@ ScyllaDB Open Source
-|   4.5                      | |x|  | |v|  | |v|  | |v|  | |x|  | |x|  | |v|  | |v|   | |x|   | |v|     | |v|   | |x|   |
-+----------------------------+------+------+------+------+------+------+------+-------+-------+---------+-------+-------+
-|   4.4                      | |x|  | |v|  | |v|  | |v|  | |x|  | |x|  | |v|  | |v|   | |x|   | |v|     | |v|   | |x|   |
-+----------------------------+------+------+------+------+------+------+------+-------+-------+---------+-------+-------+
-|   4.3                      | |x|  | |v|  | |v|  | |v|  | |x|  | |x|  | |v|  | |v|   | |x|   | |v|     | |v|   | |x|   |
-+----------------------------+------+------+------+------+------+------+------+-------+-------+---------+-------+-------+
-|   4.2                      | |x|  | |v|  | |v|  | |x|  | |x|  | |x|  | |v|  | |v|   | |x|   | |v|     | |v|   | |x|   |
-+----------------------------+------+------+------+------+------+------+------+-------+-------+---------+-------+-------+
-|   4.1                      | |x|  | |v|  | |v|  | |x|  | |x|  | |x|  | |v|  | |v|   | |x|   | |v|     | |v|   | |x|   |
-+----------------------------+------+------+------+------+------+------+------+-------+-------+---------+-------+-------+
-|   4.0                      | |x|  | |v|  | |v|  | |x|  | |x|  | |x|  | |v|  | |x|   | |x|   | |v|     | |x|   | |x|   |
-+----------------------------+------+------+------+------+------+------+------+-------+-------+---------+-------+-------+
-
-
-All releases are available as a Docker container, EC2 AMI, and a GCP image (GCP image from version 4.3). Since 
-version 5.2, the ScyllaDB AMI/Image OS for ScyllaDB Open Source is based on Ubuntu 22.04.
++----------------------------+--------------------+---------------+---------+---------------+--------------+--------------+
+| Linux Distributions        |Ubuntu              | Debian        | CentOS /| Rocky /       | Oracle Linux | Amazon Linux |
+|                            |                    |               | RHEL    | RHEL          |              |              |
++----------------------------+------+------+------+-------+-------+---------+-------+-------+-------+------+--------------+
+| ScyllaDB Version / Version | 18.04|20.04 |22.04 |  10   |  11   | 7       |   8   |   9   |  7.6  | 8.1  | 2            |
++============================+======+======+======+=======+=======+=========+=======+=======+=======+======+==============+
+|   5.4                      | |x|  | |v|  | |v|  | |v|   | |v|   | |v|     | |v|   | |v|   | |v|   | |v|  | |v|          |
++----------------------------+------+------+------+-------+-------+---------+-------+-------+-------+------+--------------+
+|   5.2                      | |x|  | |v|  | |v|  | |v|   | |v|   | |v|     | |v|   | |x|   | |x|   | |x|  | |x|          |
++----------------------------+------+------+------+-------+-------+---------+-------+-------+-------+------+--------------+
```

Comment:
Are we going to update our website once 5.4.0 is officially out? if not, then I think we should add back `5.1` since it's still supported